jest.project.js

```diff
@@ -19,7 +19,8 @@ module.exports = ({ dirname, projectMode = true }) => {
 
   return {
     ...(CI || !projectMode ? {} : { displayName: pkg.name.replace('@graphql-codegen/', '') }),
-    transform: { '^.+\\.tsx?$': 'babel-jest' },
+    transform: { '^.+\\.[tj]sx?$': 'babel-jest' },
+    transformIgnorePatterns: ['/node_modules/(?!(graphql-request)/)'],
```

Comment:
This is required because we use `graphql-request@7.1.2` in dev to run jest, which uses ESM.
Since it's already ESM, we do not need to run it through `babel-jest`.